### ISSUE TITLE
Where can I find 701 labels file?

### ISSUE THREAD

**Issue by OrkhanHI** [tinyvision/DAMO-YOLO#111]
### Before Asking

- [X] I have read the [README](https://github.com/tinyvision/DAMO-YOLO/blob/master/README.md) carefully. 我已经仔细阅读了README上的操作指引。

- [X] I want to train my custom dataset, and I have read the [tutorials for finetune on your data](https://github.com/tinyvision/DAMO-YOLO/blob/master/README.md#Finetune%20on%20your%20data) carefully and organize my dataset correctly; 我想训练自定义数据集，我已经仔细阅读了训练自定义数据的教程，以及按照正确的目录结构存放数据集。

- [X] I have pulled the latest code of main branch to run again and the problem still existed. 我已经拉取了主分支上最新的代码，重新运行之后，问题仍不能解决。

### Search before asking

- [X] I have searched the DAMO-YOLO [issues](https://github.com/tinyvision/DAMO-YOLO/issues) and found no similar questions.

### Question

I am trying to use inference on the pretrained model with 701 categories. Could you please share the file with 701 labels?

Thanks in advance.

**Comment by aitangbodan** [comment]
hello, I also need 702 labels, thank you !

**Comment by LeonNerd** [comment]
hi，感谢开源预训练模型，我也需要相关的类别标签，感谢！大数据集下的backbone对于下游任务的微调，具有更好的鲁棒性能，对于误检的抑制效果很好，

**Comment by jie311** [comment]
> hi，感谢开源预训练模型，我也需要相关的类别标签，感谢！大数据集下的backbone对于下游任务的微调，具有更好的鲁棒性能，对于误检的抑制效果很好，

请问您还有700类这个模型的pth吗，readme提供的链接无法下载

**Comment by LeonNerd** [comment]
直接wget能下

**Comment by jzx-gooner** [comment]
Same question：Where can I find 701 labels file？
